node/sync.go

```diff
@@ -421,6 +431,48 @@ func (d *Pegnetd) SnapshotPayouts(tx *sql.Tx, fLog *log.Entry, rates map[fat2.PT
 	return nil
 }
 
+// DevelopersPayouts for PIP16 sending rewards for developers
+func (d *Pegnetd) DevelopersPayouts(tx *sql.Tx, fLog *log.Entry, height uint32, heightTimestamp time.Time) error {
```

Comment:
I don't see `InsertDeveloperRewardsCoinbase()` is called in this function. @sigrlami 
Make sure if Tx is saved in DB correctly after developer rewards are processed.

Reply:
`InsertDeveloperRewardCoinbase` used in a loop over dev addresses.